client/components/domains/register-domain-step/index.jsx

```diff
@@ -760,6 +760,7 @@ class RegisterDomainStep extends Component {
 				lastDomainSearched: null,
 				isQueryInvalid: false,
 				lastQuery: cleanedQuery,
+				hideInitialQuery: false,
```

Comment:
This immediately sets `hideInitialQuery` to `false` the minute the user touches the search input field.

Reply:
What happens without it, i.e. if we the only change would be passing `new: siteSlug` param?

Reply:
Asking because I see this currently work fine with only `new` URL param in another flow: https://wordpress.com/start/domain/domain-only?new=example.org&search=yes

Reply:
Without it, the search input box would contain keyword used to populate the domain suggestions.

IMO, showing the blank search box with the default placeholder "Search..." makes it easier for user to understand this is where you enter the domain name keyword to search.

Reply:
Yep, just figured the same. Makes sense. 👍